src/data/javascript-quiz.ts

```diff
@@ -23,6 +23,15 @@ const javascriptQuiz = [
     Link:
       "https://www.freecodecamp.org/news/js-type-coercion-explained-27ba3d9a2839/",
   },
+  {
+    Question: "Which method cancels event default behavior?",
+    Answer: "preventDefault()",
+    Distractor1: "cancel()",
+    Distractor2: "preVent()",
+    Distractor3: "stop()",
+    Explanation: "The preventDefault() method of the Event interface tells the user agent that if the event does not get explicitly handled, its default action should not be taken as it normally would be.",
```

Comment:
We want to avoid using direct quotes from 3rd party resources.

```suggestion
    Explanation: "The preventDefault() method cancels the event and prevents the browser from running its default action.",
```

Reply:
Hey, @jdwilkin4 thank you for your review. I've updated the question as well as the explanation.